docs/setup/production.asciidoc

```diff
@@ -144,6 +145,13 @@ bin/kibana -c config/instance1.yml
 bin/kibana -c config/instance2.yml
 --------
 
+[float]
+[[accessing-load-balanced-kibana]]
+=== Accessing multiple load-balanced Kibana clusters
+To be able to access multiple load-balanced Kibana clusters from the same browser, the setting `xpack.security.cookieName` must be set in the configuration to avoid having conflicts between cookies from the different Kibana instances.
+
+In each cluster, Kibana instances should have the same `cookieName` value in order to achieve seamless high availability and keep the session active in case of failure from the currently used instance.
```

Comment:
```suggestion
In each cluster, {kib} instances should have the same `cookieName` value in order to achieve seamless high availability and keep the session active in case of failure from the currently used instance.
```

Reply:
@nedsir We typically use the attribute {kib} for Kibana.  Once you make the change, we can merge the doc.